EIPS/eip-7702.md

```diff
@@ -13,7 +13,7 @@ requires: 2718, 2929, 2930, 3541, 3607
 
 ## Abstract
 
-Add a new transaction type that adds a list of `[chain_id, address, nonce, y_parity, r, s]` authorization tuples. For each tuple, write a delegation designator `(0xef0000 ++ address)` to the signing account's code. All code reading operations must load the code pointed to by the designator.
```

Comment:
I believe it is a typo, as Specification has `0xef0100`

Reply:
Yes, it should also not be `0xef0000`, since by EOF terminology this means "EOF v0", which is described in the EOF EIPs as "legacy contracts" (the EOF v0), so it should definitely be something else (`0xef0100 ++ address`) would be fine, can also ditch the `00` I think?

Reply:
trailing `00` is for versioning I believe

Reply:
Oh right, that's fine and actually a great idea.